### PR TITLE
Add documentation

### LINKED ISSUE #47: vscode can't resolve complationItem/resolve

### PR DESCRIPTION
**Pull request by kitagry**
relate #47

### CODE REVIEW

**Comment by yokomotod** [comment]
sakio kosareta

**Comment by kitagry** [comment]
Could you try this branch?

**Comment by yokomotod** [comment]
tonight!

**Comment by yokomotod** [comment]
@kitagry The `textDocument/completion` for dataset/table completion succeeded and vscode can show datasets/tables 👍 

However there's still

```
[Trace - 1:36:14] Received response 'completionItem/resolve - (47)' in 0ms. Request failed: method not supported: completionItem/resolve (-32601).
[Error - 1:36:14] Request completionItem/resolve failed.
  Message: method not supported: completionItem/resolve
  Code: -32601 
```

because we don't have this method yet, so #47 is not resolved.

File: langserver/initialize.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func (h *Handler) handleInitialize(ctx context.Context, conn *jsonrpc2.Conn, req
 			HoverProvider:              true,
 			CodeActionProvider:         true,
 			CompletionProvider: &lsp.CompletionOptions{
-				ResolveProvider:   true,
+				ResolveProvider:   false,
```
Comment (kitagry) [inline langserver/initialize.go:44]: I found the cause of the issue. :cry:

Comment (yokomotod) [reply langserver/initialize.go:44]: I seeeeee.

thanks, now vscode shows no error 👍

**Pull request merged**